Assets/Scripts/Items/building/Building UI/StructurePreviewButtons.cs

```diff
@@ -19,11 +25,14 @@ public class StructurePreviewButtons : MonoBehaviour
 
         private void Awake()
         {
+            ServerInstance.Instance.TryGetComponent(out _networkManager);
```

Comment:
This isn't needed because you can just call the FishNet `InstanceFinder.NetworkManager`